2021/09/26 09:37:30 [error] 281#281: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.12, server: nginxproxymanager, request: "GET /api/ HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "192.168.1.46:81", referrer: "http://192.168.1.46:81/"
2021/09/26 09:37:45 [error] 281#281: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.12, server: nginxproxymanager, request: "GET /api/ HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "192.168.1.46:81", referrer: "http://192.168.1.46:81/"
2021/09/26 09:37:54 [error] 281#281: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.12, server: nginxproxymanager, request: "POST /api/tokens HTTP/1.1", upstream: "http://127.0.0.1:3000/tokens", host: "192.168.1.46:81", referrer: "http://192.168.1.46:81/login"
2021/09/26 10:10:27 [error] 304#304: *292 open() "/var/www/html/robots.txt" failed (2: No such file or directory), client: 45.95.147.3, server: localhost-nginx-proxy-manager, request: "HEAD /robots.txt HTTP/1.0"
